src/index.js

```diff
@@ -10,13 +12,34 @@ var _reactDom = _interopRequireDefault(require("react-dom"));
 
 var _reactHotLoader = require("react-hot-loader");
 
-var _app = _interopRequireDefault(require("./app.jsx")); // require won't find *.jsx without extension
+var _app = _interopRequireDefault(require("./app"));
+
+const { remote } = require('electron');
+const { Menu, MenuItem } = remote;
+const path = require('path');
+const JOBS_DATABASE = 'jobdb.json'
+
+let rightClickPosition = null
+const menu = new Menu();
+menu.append(new MenuItem({
+  label: 'Inspect Element',
+  click: () => { 
+    remote.getCurrentWindow().inspectElement(rightClickPosition.x, rightClickPosition.y)
+  }
+}))
```

Comment:
Should something here have a semicolon? Actually above in lines 20, 21 there also are no semicolons.

Reply:
And I'm sure they're missing in many other places. I haven't been consistent. These days semicolons are added by the interpreter as needed, but there's still much debate. If we decide on a style guide we can go from there.